Access Engineering.js

```diff
@@ -0,0 +1,333 @@
+{
+	"translatorID": "d120a8a7-9d45-446e-8c18-ad9ef0a6bf47",
+	"label": "Access Engineering",
+	"creator": "Vinoth K - highwirepress.com",
+	"target": "^https?://www.accessengineeringlibrary.com/content/(book|chapter|case-study|video|calculator|tutorial)",
```

Comment:
Regexify:

```suggestion
	"target": "^https?://www\\.accessengineeringlibrary\\.com/content/(book|chapter|case-study|video|calculator|tutorial)",
```

Reply:
Updated above same regular expression in code.